### PR TITLE
Override overhaul

### PR DESCRIPTION
**Pull request by benghaziboy**

### CODE REVIEW

**Comment by benghaziboy** [comment]
@mparent61

**Comment by benghaziboy** [comment]
@MichaelButkovic

**Comment by MichaelButkovic** [comment]
:+1:

File: bulbs/contributions/models.py [hunk 1]
```diff
@@ -93,21 +93,29 @@ def get_rate(self):
...
-            else:
+        # Get contribution specific overrides first.
+        if self.role.payment_type in [0, 1] and self.override_contribution.exists():
+            return self.override_contribution.first().rate
+
+        override_profile = self.contributor.overrides.filter(role=self.role).first()
+        if not override_profile:
+            return None
+
+        elif self.role.payment_type == 0 and override_profile.override_flatrate.exists():
```
Comment (mparent61) [inline bulbs/contributions/models.py]: use enums for all of these payment_type integers

File: bulbs/contributions/models.py [hunk 2]
```diff
@@ -116,40 +124,43 @@ def _get_pay(self):
         rate = self.get_rate()
         if isinstance(rate, HourlyRate):
             minutes_worked = float(getattr(self, 'minutes_worked', 0))
-            return ((float(rate.rate) / 60) * float(minutes_worked))
+            return calculate_hourly_pay(rate.rate, minutes_worked)
```
Comment (mparent61) [inline bulbs/contributions/models.py]: nice helper function

**Pull request merged**